### PR TITLE
prelude.0.1

### PR DESCRIPTION
**Pull request by zapashcanon**

### CODE REVIEW

File: packages/prelude/prelude.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "OCaml library hiding stuff from the stdlib"
+description:
+  "Prelude is an OCaml library hiding a lot of stuff from the stdlib."
+maintainer: ["Léo Andrès <contact@ndrs.fr>"]
+authors: ["Léo Andrès <contact@ndrs.fr>"]
+license: "AGPL-3.0-or-later"
+tags: ["prelude" "stdlib" "safe" "exception" "layer" "hide"]
+homepage: "https://git.zapashcanon.fr/zapashcanon/prelude"
+doc: "https://doc.zapashcanon.fr/prelude"
```
Comment (avsm) [inline packages/prelude/prelude.0.1/opam:10]: This is a 404

Comment (zapashcanon) [reply packages/prelude/prelude.0.1/opam]: Fixed by removing the link.

**Comment by raphael-proust** [comment]
Thanks for the contribution!!

**Comment by zapashcanon** [comment]
Thanks!

**Pull request merged**